scripts/ci/wasm_unpack.js

```diff
@@ -0,0 +1,235 @@
+(function(source, uncompressedSize) {
+	var base64DecToArr = (Buffer !== undefined && typeof Buffer.from === "function") ?
+	function (sB64Enc) { return Buffer.from(sB64Enc, 'base64'); } : function (sB64Enc) {
+		/*\
+		|*|
+		|*|  Base64 / binary data / UTF-8 strings utilities (#1)
+		|*|
+		|*|  https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/Base64_encoding_and_decoding
+		|*|
+		|*|  Author: madmurphy
+		|*|
+		\*/
+		/* Array of bytes to base64 string decoding */
+		function b64ToUint6 (nChr) {
+			return nChr > 64 && nChr < 91 ?
+				nChr - 65
+				: nChr > 96 && nChr < 123 ?
+				nChr - 71
+				: nChr > 47 && nChr < 58 ?
+				nChr + 4
+				: nChr === 43 ?
+				62
+				: nChr === 47 ?
+				63
+				:
+				0;
+		}
+
+		var nInLen = sB64Enc.length,
+			nOutLen = nInLen * 3 + 1 >>> 2,
+			aBytes = new Uint8Array(nOutLen);
+
+		for (var nMod3, nMod4, nUint24 = 0, nOutIdx = 0, nInIdx = 0; nInIdx < nInLen; nInIdx++) {
+			nMod4 = nInIdx & 3;
+			nUint24 |= b64ToUint6(sB64Enc.charCodeAt(nInIdx)) << 18 - 6 * nMod4;
+			if (nMod4 === 3 || nInLen - nInIdx === 1) {
+				for (nMod3 = 0; nMod3 < 3 && nOutIdx < nOutLen; nMod3++, nOutIdx++) {
+					aBytes[nOutIdx] = nUint24 >>> (16 >>> nMod3 & 24) & 255;
+				}
+				nUint24 = 0;
+			}
+		}
+		return aBytes;
+	}
+	function lz4Decompress (src, dst) {
+		// lz4.js - An implementation of Lz4 in plain JavaScript.
+		// Original source: https://github.com/Benzinga/lz4js
+
+		// Reads a 32-bit little-endian integer from an array.
```

Comment:
I significantly simplified it anyways (e.g. I stripped the encoding only parts and merged it into a single function in a single file)... so updating it would be manual effort anyways, we can't just drop-in replace it with a new version...
Given that, I'd argue that having it in a single file in which you can look through the entire process makes it easier to read... but I'm also fine with splitting it up somehow.

Reply:
Actually this makes having it in a separate file even better. You could commit the original and then apply your changes on it (unless it was huge and you just removed 90% of it :)). It would make it easy to see what was changed.

Reply:
Originally it's three separate files that I merged into one, though :-)...
But yeah - the first question here is actually whether it's worth doing anything like that.
Are we fine with https://www.npmjs.com/package/lz4js and https://github.com/Benzinga/lz4js/blob/master/package.json stating ``ISC`` as license without a full license file anywhere :-)? Also should we then mention that in ``solc --license`` :-)?

It's not too hard to just rewrite the decompression logic from spec (even more so if we verify that it works on the single input we need as discussed in the other comments), so I could also just do that...

Reply:
> Originally it's three separate files that I merged into one, though :-)...

Even more reason to keep them separate :)

> Also should we then mention that in `solc --license`

AFAIK `solc --license` is only for stuff being part of the `solc` binary so I guess not unless there's also some kind of `license()` function in `soljson.js` that can give you this info - then yes.

> It's not too hard to just rewrite the decompression logic from spec (...), so I could also just do that...

Sounds like an overkill just for this. Even if it's relatively easy, details always take time to get right. Probably not worth it unless we absolutely can't use the lib you found.